### DISCUSSION TITLE
Command lang:sync

### DISCUSSION THREAD

**Discussion by masterix21** [Laravel-Lang/common#47] [Q&A] [answered]
Hi. To use the translations, you should manually copy the files in the resources/lang directory.

What about a command like `lang:sync <languages>`?

1. If the translation doesn't exist, it will copy the file.
2. If the file exists, the command will open it and keep synced with our new translations, preserving the custom user translations (useful with JSON translation files).

**Comment by caouecs** [comment, marked as answer]
## Projects based on this package

* [LaravelLang](https://github.com/ARCANEDEV/LaravelLang) by [*@arcanedev-maroc*](https://github.com/ARCANEDEV) - Translations manager and checker for Laravel.
* [Laravel-lang](https://github.com/overtrue/laravel-lang) by [*@overtrue*](https://github.com/overtrue) - Command to add languages in your project.
* [Lang Publisher](https://github.com/andrey-helldar/laravel-lang-publisher) by [*@andrey-helldar*](https://github.com/andrey-helldar) - Easy installation and update of translation files for your project.

**Reply by andrey-helldar** [reply]
@masterix21, this is already implemented in my [laravel-lang-publisher](https://github.com/andrey-helldar/laravel-lang-publisher/blob/main/README.md#update-locales) package.

`php artisan lang:update` for the php files.
`php artisan lang:update --json` for the json files.

**Reply by masterix21** [reply]
Great. Thanks.
